### DISCUSSION TITLE
Changing the source branch when creating a branch from an issue no longer works

### DISCUSSION THREAD

**Discussion by nicole-obrien** [community/community#19344] [Projects and Issues] [answered]
Currently when trying to create a branch for an issue, if you change the branch source so that it is no longer the default branch it does not work. 

By does not work, I mean that when I fetch the branch and switch to it using the copyable code snippet presented it is a branch based of the default branch and not the branch that I selected.

**Comment by ansballard** [comment, marked as answer]
Hi @nicole-obrien! This bug should be resolved, it was fixed as of ~3 days ago. Feel free to reach back out if you're still able to reproduce it!
